test/slash_command.spec.ts

```diff
@@ -86,103 +90,206 @@ let mockedGithubHelper = new GithubHelper(mockOctokit, "unit_test_owner", "unit_
 
 beforeEach(() => {
     //jest.resetAllMocks();
-})
-
-test('handle /help', async () => {
-    // Arrange
-    const prInfo: PullRequestInfo = {
-        branch: "",
-        sha1: ""
-    }
-    const command = "/help"
-    const commentId = 534
-
-    // Act
-    await handleSlashCommand(
-        mockedTfcApi,
-        mockedTfcCli,
-        mockedGithubHelper,
-        prInfo,
-        commentId,
-        command
-    );
-
-    // Assert
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls.length).toBe(2);
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls[0][0].content).toBe("eyes");
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls[0][0].comment_id).toBe(commentId);
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls[1][0].content).toBe("rocket");
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls[1][0].comment_id).toBe(commentId);
-    expect(mockOctokit.rest.issues.createComment.mock.calls.length).toBe(1);
-    expect(mockOctokit.rest.issues.createComment.mock.calls[0][0].body).toContain(HELP_TEXT);
-    expect(mockExec.mock.calls[0][0]).toContain("terraform init")
 });
 
-test('handle /deploy', async () => {
-    // Arrange
-    const prInfo: PullRequestInfo = {
-        branch: "test-branch",
-        sha1: "abc1234"
-    }
-    const command = "/deploy"
-    const commentId = 534
-
-    // Act
-    await handleSlashCommand(
-        mockedTfcApi,
-        mockedTfcCli,
-        mockedGithubHelper,
-        prInfo,
-        commentId,
-        command
-    );
-
-    // Assert
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls.length).toBe(2);
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls[0][0].content).toBe("eyes");
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls[0][0].comment_id).toBe(commentId);
-
-    expect(mockExec.mock.calls[0][0]).toContain("terraform init")
-    expect(mockExec.mock.calls[1][0]).toContain("terraform apply")
-
-    expect(mockOctokit.rest.issues.createComment.mock.calls.length).toBe(1);
-    expect(mockOctokit.rest.issues.createComment.mock.calls[0][0].body).toContain(":::environment created:::");
-
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls[1][0].content).toBe("rocket");
-    expect(mockOctokit.rest.reactions.createForIssueComment.mock.calls[1][0].comment_id).toBe(commentId);
+describe('[Terraform Cloud]', () => {
```

Comment:
these are all the same tests that we had before but I isolated them into a `describe` function to scope the api configuration/mocking